client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -78,10 +116,11 @@ where
         from_address: &ExtendedAddr,
         value: Coin,
         attributes: StakedStateOpAttributes,
+        nonce: Nonce,
```

Comment:
can be removed from high-level APIs

Reply:
yes , it's removed.
will be automatically fetched from core